Update data_plane_template.md (#34086)
Azure/azure-rest-api-specs: .github/PULL_REQUEST_TEMPLATE/data_plane_template.md

[Swagger-Suppression-Process](https://aka.ms/pr-suppressions) 
to get approval.

### Release planner

> A [release plan](https://aka.ms/azsdkdocs/release-plans) should have been created, please do so if you have not. A release plan well help guide you through the REST API and SDK creation process. 

Reviewer: ```suggestion
A [release plan](https://aka.ms/azsdkdocs/release-plans) should have been created. If not, please create one as it will help guide you through the REST API and SDK creation process. 
```
